modules/latentsvm/test/test_latentsvmdetector_cascade.cpp

```diff
@@ -127,13 +127,14 @@ bool compareResults( const std::vector<lsvm::LSVMDetector::ObjectDetection>& cal
 
 void LatentSVMDetectorCaskadeTest::run( int /* start_from */)
 {
-    std::string img_path_cat = std::string(ts->get_data_path()) + "cat.png";
-    std::string img_path_cars = std::string(ts->get_data_path()) + "cars.png";
+	std::string test_data_path = ts->get_data_path() + "latentsvmdetector/";
```

Comment:
Yes, thanks for pointing this out. By the way, can we change `Caskade` into `Cascade`? Isn't it a typo?